contracts/multisig-prover/src/events.rs

```diff
@@ -1,32 +1,76 @@
+use crate::types::BatchId;
+use axelar_wasm_std::event;
+use connection_router::state::{ChainName, CrossChainId};
 use cosmwasm_std::Uint64;
 use serde_json::to_string;
 
-use crate::types::BatchId;
-
 pub enum Event {
     ProofUnderConstruction {
+        destination_chain: ChainName,
         command_batch_id: BatchId,
         multisig_session_id: Uint64,
+        msg_ids: Vec<CrossChainId>,
     },
 }
 
 impl From<Event> for cosmwasm_std::Event {
     fn from(other: Event) -> Self {
         match other {
             Event::ProofUnderConstruction {
+                destination_chain,
                 command_batch_id,
                 multisig_session_id,
+                msg_ids,
             } => cosmwasm_std::Event::new("proof_under_construction")
+                .add_attribute(
+                    "destination_chain",
+                    event::attribute_value(&destination_chain)
```

Comment:
What I don't like about this is that, there's absolutely nothing there to require people to use it but nothing else.

Reply:
I'd prefer to catch this with something like dylint instead of requiring us wrapping their Event and basically reimplementing it. Also I'd like to tackle that separately from this PR